src/Command/ShowCommand.php

```diff
@@ -29,14 +29,6 @@ class ShowCommand extends ReflectingCommand
     private $lastException;
     private $lastExceptionIndex;
 
-    /**
-     * @param string|null $colorMode (deprecated and ignored)
-     */
-    public function __construct($colorMode = null)
```

Comment:
This is intentionally preventing `$colorMode` from being passed to `Symfony\Component\Console\Command\Command`, which expects an optional `$name` instead :-/